src/ray/core_worker/test/actor_task_submitter_test.cc

```diff
@@ -667,6 +668,9 @@ TEST_P(ActorTaskSubmitterTest, TestActorRestartFailInflightTasks) {
   ASSERT_TRUE(worker_client_->ReplyPushTask(Status::OK()));
   // Task 3 replied with error.
   ASSERT_TRUE(worker_client_->ReplyPushTask(Status::IOError("")));
```

Comment:
How do we know these responses are replying to task 2 and task 3, respectively? The call doesn't pass in the task spec or index

Reply:
Because the callbacks are stored in a vector in the submission order. Will update it to be more explicit.